pkg/apis/nfs.rook.io/v1alpha1/types.go

```diff
@@ -82,6 +82,10 @@ type NFSServerList struct {
 
 // NFSServerSpec represents the spec of NFS daemon
 type NFSServerSpec struct {
+	// ServiceAccountName is the name of the ServiceAccount to use to run this nfs server pod.
+	// More info: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```

Comment:
Instead of adding the serivce account name to the CR settings, I would recommend just requiring the service account with a specific name such as `rook-nfs-server`. In the ceph provider we also make assumptions on the service account names rather than exposing them as settings. We just haven't found it necessary.

Reply:
make sense, if so i would prefer the `default` service account instead of creating a new service account since it's already scoped by namespace. what do you think @travisn